src/fmt.zig

```diff
@@ -2432,7 +2492,7 @@ test "Syntax grab bag" {
         \\main! : List(String) -> Result({}, _)
         \\main! = |_| { # Yeah I can leave a comment here
         \\    world = "World"
-        \\    number = 123
+        \\    var number = 123
```

Comment:
Is it enforced that the ident must end with a `_` or will that be checked in Can?